spec/teacher_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../teacher'
+
+describe Teacher do
+  before(:each) do
+    @teacher = Teacher.new('Developer', 5, 'Tiwa', parent_permission: true)
+  end
+
+  it 'Should be create a student name' do
+    expect(@teacher.name).to eq 'Tiwa'
+  end
+
+  it 'Should be create a student name' do
+    expect(@teacher.specialization).to eq 'Developer'
+  end
+end
```

Comment:
- [ ] Same here, there should be unit tests for all the methods in all the classes, Kindly make sure the `can_use_services?` class method has (at least) an 'it' block 👇🏻 check your teacher,rb file

```ruby
   def can_use_services?
    true
  end
```

Reply:
Allready implemented it! Thanks for the feedback! :smile: